db/migrate/20170808080710_move_enabled_to_repo_services.rb

```diff
@@ -0,0 +1,19 @@
+class MoveEnabledToRepoServices < ActiveRecord::Migration[5.1]
+
+  def up
+    add_column :repositories, :installer_updates, :boolean, null: false, default: false
+    add_column :repositories, :mirroring_enabled, :boolean, null: false, default: false
+    add_column :repositories, :local_path, :string, null: false
```

Comment:
You are not supposed to have repos without local paths anymore.
I should probably add some code to invoke `make_local_path` during up migration.